index.js

```diff
@@ -458,6 +466,18 @@ module.exports = {
     'operator-linebreak': 'warn',
     // enforce padding within blocks
     'padded-blocks': ['warn', 'never'],
+    // require or disallow padding lines between statements
+    'padding-line-between-statements': ['warn',
+      { blankLine: 'always', prev: '*', next: 'return' },
+      { blankLine: 'always', prev: ['const', 'let', 'var'], next: '*' },
+      {
+        blankLine: 'any',
+        prev: ['const', 'let', 'var'],
+        next: ['const', 'let', 'var']
+      },
+      { blankLine: 'always', prev: 'directive', next: '*' },
+      { blankLine: 'any', prev: 'directive', next: 'directive' }
+    ],
```

Comment:
The release notes/CHANGELOG lists deprecated rules along with information about their replacements.